src/werkzeug/serving.py

```diff
@@ -834,6 +841,7 @@ def log_startup(sock):
                 port,
                 quit_msg,
             )
+        map(startup_messages, lambda msg: _log(msg["type"], msg["message"]))
```

Comment:
`map` is  NOT meant to be used as a replacement for a `for` loop, and I'd say having side effects in the callback of `map` is always a bad idea. It also does not work in Python 3 since map returns an iterator there.

Reply:
I'll use a for loop.